assets/models/system/openai-clip-image-text-embeddings-vit-large-patch14-336/description.md

```diff
@@ -0,0 +1,109 @@
+
+> The above summary was generated using ChatGPT. Review the <a href="https://huggingface.co/openai/clip-vit-large-patch14-336" target="_blank">original-model-card</a> to understand the data used to train the model, evaluation metrics, license, intended uses, limitations and bias before using the model.
+
+### Inference samples
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-sdk-image-text-embeddings" target="_blank">image-text-embeddings-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-cli-image-text-embeddings" target="_blank">image-text-embeddings-online-endpoint.sh</a>
+Batch|<a href="https://aka.ms/azureml-infer-batch-sdk-image-text-embeddings" target="_blank">image-text-embeddings-batch-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-batch-cli-image-text-embeddings" target="_blank">image-text-embeddings-batch-endpoint.sh</a>
+
+### Sample inputs and outputs (for real-time inference)
+
+#### Sample input for image embeddings
+
+```json
+{
+   "input_data":{
+      "columns":[
+         "image", "text"
+      ],
+      "index":[0, 1],
+      "data":[
+         ["image1", ""],
+         ["image2", ""]
+      ]
+   }
+}
+```
+Note: "image1" and "image2" should be publicly accessible urls or strings in `base64` format
+
+#### Sample output
+
+```json
+[
+    {
+        "image_features": [-0.92, -0.13, 0.02, ... , 0.13],
+    },
+    {
+        "image_features": [0.54, -0.83, 0.13, ... , 0.26],
+    }
+]
+```
+Note: returned embeddings have dimension 768 and are not normalized
+
+#### Sample input for text embeddings
+
+```json
+{
+   "input_data":{
+      "columns":[
+         "image", "text"
+      ],
+      "index":[0, 1],
+      "data":[
+         ["", "sample text 1"],
+         ["", "sample text 2"]
+      ]
+   }
+}
+```
+
+#### Sample output
+
+```json
+[
+    {
+        "text_features": [0.42, -0.13, -0.92, ... , 0.63],
+    },
+    {
+        "text_features": [-0.14, 0.93, -0.15, ... , 0.66],
+    }
+]
+```
+Note: returned embeddings have dimension 768 and are not normalized
+
+#### Sample input for image and text embeddings
+
+```json
+{
+   "input_data":{
+      "columns":[
+         "image", "text"
+      ],
+      "index":[0, 1],
+      "data":[
+         ["image1", "sample text 1"],
+         ["image2", "sample text 2"]
+      ]
+   }
+}
+```
+Note: "image1" and "image2" should be publicly accessible urls or strings in `base64` format
+
+#### Sample output
+
+```json
+[
+    {
+        "image_features": [0.92, -0.13, 0.02, ... , -0.13],
+        "text_features": [0.42, 0.13, -0.92, ... , -0.63]
+    },
+    {
+        "image_features": [-0.54, -0.83, 0.13, ... , -0.26],
+        "text_features": [-0.14, -0.93, 0.15, ... , 0.66]
+    }
+]
+```
+Note: returned embeddings have dimension 768 and are not normalized
```

Comment:
768 -> 1024

Reply:
<img width="820" alt="image" src="https://github.com/Azure/azureml-assets/assets/38021489/f5e9b29d-1fc0-42af-be15-0157322c8877">
The embedding dimension is 768. I have also verified this by inspecting the embeddings returned from the endpoint.